### ISSUE TITLE
oppo_preloader

### ISSUE THREAD

**Issue by qianmo-xw** [MTK-bypass/bypass_utility#32]
Models like OPPO A83 A1 use MT6763 cpu, and new security is enabled after 2018, which causes the device to display the driver as OPPO Preloader after pressing the volume and insert the data cable, and after installing the driver, it displays as MediaTek Preloader USB  VCOM, but the PID and VID of the driver are not the same as the ordinary mtk device. The VID of the device is 22D9 and the PID is 0006. Bypass_utility cannot detect the device. When I change the pid and vid in device.py in src,  I got such an error. I think the port is still a bootrom port, so can we support oppo's preloader port?

![6b9efcaad5fa7e14](https://user-images.githubusercontent.com/83547396/116819470-cc906980-aba2-11eb-82d1-6fde90a9bb04.png)

**Comment by chaosmaster** [comment]
The device needs to be in bootrom, not in preloader.
You can try changing just the VID to see if the preloader can be crashed.

**Comment by qianmo-xw** [comment]
> 该设备必须位于bootrom中，而不是预加载器中。
> 您可以尝试仅更改VID，以查看预加载器是否可能崩溃。

I have tried to change only the VID, bypass_utility prompts the preloader device and then tries to crash, and then the device is disconnected. If I don’t press the key, the device will turn on directly. If I hold down the volume, the device will enter the preloader again and repeat the process.  I don’t know what I did wrong, tomorrow I will post the stickers and provide the log.  Or is there a way to convert the preloader port to a bootrom port?
